public-api/src/main/resources/public-api.yaml

```diff
@@ -61,15 +61,15 @@ paths:
     get:
       tags:
         - dataBrowser
-
       description: Gets the cdr versions
       operationId: "getCdrVersionUsed"
+      security: []
```

Comment:
![image](https://user-images.githubusercontent.com/21062724/55571124-7d796100-56ca-11e9-8da5-d4a94666e1f7.png)
This is to deal with this error.
Not sure if it helps though.
I do not see this error on my local server.